addons/mail/static/src/models/channel.js

```diff
@@ -185,6 +185,10 @@ registerModel({
             compute: '_computeCorrespondentOfDmChat',
             inverse: 'dmChatWithCurrentPartner',
         }),
+        /**
+         * FIXME: should rather be on ChannelMember.
+         */
```

Comment:
Remove FIXME, the modelling will be changed when needed, and it's more appropriate to actually rename field and/or move it to another model when it feels best.